scss/6-components/_search.scss

```diff
@@ -75,3 +75,18 @@
     }
   }
 }
+
+/* stylelint-disable plugin/selector-bem-pattern */
+.cads-header__search-row {
```

Comment:
If possible we should avoid touching other component classes, could we add a `cads-header__search-form` wrapper so we could write:

```
.cads-header__search-form {
    margin-left: auto;

    @include cads-media-breakpoint-only(md) {
      width: 70%;
    }

    @include cads-media-breakpoint-only(lg) {
      width: 55%;
    }
}
```

Avoids the need for the `stylelint-disable` then

Reply:
well that's a good idea 👍